openassessment/xblock/static/js/src/studio/oa_edit_assessment.js

```diff
@@ -621,3 +621,104 @@ OpenAssessment.EditExampleBasedAssessmentView.prototype = {
     validationErrors: function() { return []; },
     clearValidationErrors: function() {}
 };
+
+/**
+ Interface for editing staff assessment settings.
+
+ Args:
+ element (DOM element): The DOM element representing this view.
+
+ Returns:
+ OpenAssessment.EditStaffAssessmentView
+
+ **/
+OpenAssessment.EditStaffAssessmentView = function(element) {
+    this.element = element;
+    this.name = "staff-assessment";
+
+    // Configure the toggle checkbox to enable/disable this assessment
+    new OpenAssessment.ToggleControl(
+        $("#include_staff_assessment", this.element),
+        $("#staff_assessment_description", this.element),
+        $("#staff_assessment_description", this.element), //open and closed selectors are the same!
+        new OpenAssessment.Notifier([
+            new OpenAssessment.AssessmentToggleListener()
+        ])
+    ).install();
+};
+
+OpenAssessment.EditStaffAssessmentView.prototype = {
+
+    /**
+     Return a description of the assessment.
+
+     Returns:
+     object literal
+     **/
+    description: function() {
+        return {
+            required: this.isEnabled(),
```

Comment:
`description` gets serialized and passed to the backend as the requirements for a given step. Since staff assessment is unique in that the presence of the step does not tell us any information about requirements, we need to explicitly put that information here.

Stack trace, if anyone is curious:
* `description` value is grabbed here: https://github.com/edx/edx-ora2/blob/1583a5d2fe6379dd10512502094d465b0ca20c67/openassessment/xblock/static/js/src/studio/oa_edit_settings.js
* that function if called from here: https://github.com/edx/edx-ora2/blob/1583a5d2fe6379dd10512502094d465b0ca20c67/openassessment/xblock/static/js/src/studio/oa_edit.js#L202
* the return value is passed to here: https://github.com/edx/edx-ora2/blob/1583a5d2fe6379dd10512502094d465b0ca20c67/openassessment/xblock/static/js/src/oa_server.js#L450
* which updates the xblock definitions here: https://github.com/edx/edx-ora2/blob/2bd3d4017491c30e4fdd077829db69b2b8e33d9c/openassessment/xblock/studio_mixin.py#L148